docs/docs/reference/coordinator.md

```diff
@@ -1,5 +1,1223 @@
-import ApiDocMdx from '@theme/ApiDocMdx';
+# Coordinator client API
 
-# Coordinator Client API
+The Coordinator exposes an HTTP-REST interface, referred to as the Client API.
 
-<ApiDocMdx id="coordinator-swagger-json" />
+Responses of the API follow the [JSend specification](https://github.com/omniti-labs/jsend).
+This means all endpoints return a JSON object with a `status` field that's either `success`, `fail`, or `error`:
+
+* In case of `success`, the response will contain a `data` field with the actual response data.
+* In case of `error`, the response will contain a `message` field with an error message.
```

Comment:
```suggestion
* In case of `success`, the response contains a `data` field with the actual response data.
* In case of `error`, the response contains a `message` field with an error message.
```